requests-for-comments/2023-17-produchog.md

```diff
@@ -0,0 +1,92 @@
+# RFC: The future of HogMail is... ProductHog
+
+HogMail was launched in April as a mostly PostHog orientated news drop. It’s since evolved into a more product growth-themed newsletter hybrid. We send it every two weeks – this won’t change.
+
+This RFC sets out some ideas and discussion points for taking HogMail to the next level by treating it as a distinct brand and product, created by PostHog.
+
+Most recent issue for context + feedback: [https://mailchi.mp/posthog/hogmail-15794324](https://mailchi.mp/posthog/hogmail-15794324)
+
+## Where we are now
+
+- Our subscriber list has grown 49% since April – 662 to 987 as of writing.
+- Open rates have increased from ~43% at launch to ~47% to 50% now. Based on my research, this is above average but not market-leading for a newsletter of our type.
+- Each issue is split into three sections: new PostHog articles, curated articles with commentary, misc. interesting reads.
+- Most signups come via CTAs on the website and promotion in The Array
+- At the current rate of growth, we’d reach 1,500 subscribers by the end of 2023
+
+## Where we want to be
+
+- 3,000 subs by end of 2023 – a 200% increase on today
+- People sharing and recommending newsletter on social media
+- Be a newsletter that’s perceived as a desirable source of promotion for content creators
+
+## Proposal: Rebrand to ‘ProductHog’ and launch on Substack
+
+- The current branding was a simple way to get started, but is limiting.
+- ‘ProductHog’ implies what the newsletter is about, while still drawing a connection between it and PostHog.
+- Actions: create website for ProductHog (we own the domain) and launch dedicated social channels.
+
+### Why Substack?
+
+Currently the newsletter is on Mailchimp, but we’ve recently migrated other email ops to [Customer.io](http://Cusomter.io). 
+
+I’m proposing Substack because:
+
+- It’s a popular platform people know and like
+- It would make this feel distinct from other “marketing emails”
+- Allows us to tap into the Substack network effects to aid growth
+
+The downsides of Substack:
+
+- We’ll have less control over branding and appearance
```

Comment:
do you think if it's a non-monetized substack, then substack won't internally promote it as much?

Reply:
Honestly, I don't know, and this is a good point. I don't have enough experience with Substack to say.

Another issue with Substack is our model would still, largely, be about summarizing content in the newsletter and sending people to our website to read full posts. I'm not sure how well this approach sits as well on Substack, where readers typically expect whole posts to be published on Substack itself.

Reply:
I don't think so. The promotion comes from the feeds which aren't algorithmic yet and recommendations which also aren't algorithmic yet. A lot of the benefit also comes from all the growth-driven UI they automatically include.

Related: there is something there are doing now where readers can "pledge" how much they would give if you started making it paid.

Reply:
[From a big Substack writer](https://thezvi.substack.com/p/looking-back-on-posts-from-2022): "No one clicks on links, maybe ~25% of users click even one in a giant post."